client/extensions/woocommerce/index.js

```diff
@@ -151,15 +155,23 @@ function addStorePage( storePage, storeNavigation ) {
 	} );
 }
 
-function createStoreNavigation( context, next ) {
-	const state = context.store.getState();
-	const selectedSite = getSelectedSite( state );
+function mapStateToProps( state ) {
+	const currentUser = getCurrentUser( state );
+	const selectedSiteId = getSelectedSiteId( state );
+	const isSingleSite = !! selectedSiteId || currentUser.site_count === 1;
```

Comment:
Maybe worth creating a new selector or helper function so both this and [and the sidebar code](https://github.com/Automattic/wp-calypso/blob/cafb1a709743d0f4f589def09943373ec53513ba/client/my-sites/sidebar/sidebar.jsx#L599) could take use of it?